### ISSUE TITLE
Darker Theme: Low contrast on excluded folders in tree

### ISSUE THREAD

**Issue by nazrhyn** [mallowigi/material-theme-jetbrains-eap#78]
![image](https://user-images.githubusercontent.com/1672405/28135744-4ace51da-6715-11e7-9454-ea2410c59921.png)

In this situation, the contrast between the gray folder name text and the blue highlight is pretty low. It's fairly hard to read.

I know the folders are excluded (and that's why the name is grayed back and the folder icon is red), but I'm not sure why there is a permanent row highlight.

**Comment by mallowigi** [comment]
FYI, for the Darker theme there is a predefined "File Colors" in Appearance > File Colors. Just put Darker above all else and you should have a better option.

As for the gray color, this is because this is the color of ignored files. I will look at it.

**Comment by mallowigi** [comment]
try the latest release (hotfix) and tell me if it happens again (grayed file). I didn't test thoroughly bcz im pretty tired but I will in a short while.

**Comment by nazrhyn** [comment]
Ah, cool. That does look a lot better; thanks.

![image](https://user-images.githubusercontent.com/1672405/28141009-ba46fb26-6728-11e7-8ccf-a7738b601dd8.png)

Good eyes noticing where that blue came from :wink:.

**Comment by nazrhyn** [comment]
Looks the same after the hotfix:

![image](https://user-images.githubusercontent.com/1672405/28141381-55529ef8-672a-11e7-99e3-893861bcc443.png)

But, TBH, the contrast is way better once those file colors get ordered with darker first. That's kind of a fiddly thing to have to know to do, though, isn't it? Are there any alternatives?

Get some sleep! I'll be here tomorrow :+1:.

**Comment by mallowigi** [comment]
I agree, and I wrote a whole section in the README, because I really don't have a lot of options here, because I can't change the user's settings programmatically.

Think that you could have your own file colors and don't need to use the ones I'm providing.

TL;DR: read the README ;)

--
Elior

**Comment by nazrhyn** [comment]
Oh boy. Now I'm _that guy_.
